CMakeLists.txt

```diff
@@ -279,5 +279,11 @@ elseif(${CMAKE_TARGET_ARCHITECTURE} STREQUAL "ppc64")
             "${CMAKE_UPPER_WORKSPACE_NAME}_HAS_POWER8")
 endif()
 
-include(CMTest)
-cm_add_test_subdirectory(test)
+if(BUILD_TESTS)
```

Comment:
Don't change it. After revert remove `include(CMTest)` from _test/CMakeLists.txt_
https://github.com/NilFoundation/crypto3/issues/146

Reply:
Once I revert this, hash_* targets are not available when building from Solana project. I use the following cmake:


cmake .. -DCMAKE_BUILD_TYPE=Debug  -DCMAKE_CXX_FLAGS=-ggdb  -DBUILD_SHARED_LIBS=FALSE -DBUILD_WITH_SANITIZE=FALSE  -DZK_PLACEHOLDER_PROFILING_ENABLED=TRUE -DBLUEPRINT_PLACEHOLDER_PROOF_GEN=True  -DBUILD_TESTS=TRUE